gensim/models/_fastldamodel.pyx

```diff
@@ -0,0 +1,256 @@
+from __future__ import division
```

Comment:
This file is needed right now (after last changes where you move all "slowest" function to cython)?

Reply:
Also, `.pyd` file (windows compiled) should be here (only `pyx` and `c`)

Reply:
no, not need.  Left it in just for comparison, but removed it now.